src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Represents exceptions unique to the application.
+ */
+public class DukeException extends Exception {
```

Comment:
Could this class be abstracted more into separate kinds of exceptions to be thrown ?